grass7/raster/r.confusionmatrix/testsuite/test_r_confusionmatrix.py

```diff
@@ -0,0 +1,60 @@
+"""
+Name:       r.confusionmatrix test
+Purpose:    Tests r.confusionmatrix input parsing.
+            Uses NC Basic data set.
```

Comment:
```suggestion
            Uses NC full sample data set.
```
(in NC basic there is "landuse" but not "landclass")